samba-dc/entrypoint.sh

```diff
@@ -23,6 +23,7 @@ function domain_controller_role ()
     samba -F --debug-stdout "${extra_args[@]}" &
     chronyd -d -x &
     wsdd -i "${IPADDRESS}" -d "${NBDOMAIN}" &
+    syslog-ng -F --no-caps &
```

Comment:
The --no-caps options seems the default, maybe we do not need to specify it:
https://linux.die.net/man/8/syslog-ng

Reply:
The doc always lies ;)
Ok!